src/data/defaults.txt

```diff
@@ -1106,6 +1106,7 @@ user	peteMotorbikeMuffler
 user	peteMotorbikeSeat
 user	photocopyMonster
 user	pieStuffing
+user	pingpongSkill	0
```

Comment:
wiki says "Is reset on ascension to 1.".  Might not matter if we can always look it up.  Is 1 a better default?

Reply:
Wiki had nada about this when I submitted.

If 1 is the default, yes.
And if it resets, that needs to be done as well.

Reply:
I took a character with no ping-pong skill.
Nothing was in the quest log.
He got a ping-pong paddle.
Nothing was in the quest log.
He used on ping-pong ball.
```
You have achieved a skill level of 1 at playing ping-pong.
```
I would be surprised if your 1st skill level carried across ascension.
Should I believe the Wiki?

Reply:
I'm resetting to 0 on ascension.
I made it read ping-pong skill from the quest log.
If it really is one on ascension, that would be a first: reset on ascension to something which is NOT your initial state.
Whatever. We will correct when you look at quest log.